test-files/cases/unifying/poi-polygon/poi-polygon-auto-merge-13/Expected.osm

```diff
@@ -0,0 +1,62 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
This tests and poi-polygon-auto-merge-14 test a situation where merging of a poi into a poly was occurring and shouldn't have been due to an address parsing bug pre-existing before these changes.